packages/block-library/src/image/edit.native.js

```diff
@@ -261,6 +294,56 @@ class ImageEdit extends React.Component {
 			</BottomSheet>
 		);
 
+		//Used for display of Inspector Controls
+		const getSizeSlugDisplay = ( sizeSlugValue ) => {
```

Comment:
Having so many functions defined in the render method isn't good practice. This could probably be moved outside the class or as a class method if we get the `sizeSlugValue` from props.

Reply:
Moved to ImageSizePicker in 7983f04c71df03900122d95e885749c6eb1665c2